src/leakybucket.h

```diff
@@ -32,13 +32,15 @@ class CLeakyBucket
     static CClock clock;
     boost::chrono::time_point<CClock> lastFill;
 
-    // This function is called internally to fill the leaky bucket based on the time difference between now and the last time the function was called.
+    // This function is called internally to fill the leaky bucket based on the time difference between now and the last
+    // time the function was called.
     void fillIt()
     {
         boost::chrono::time_point<CClock> now = clock.now();
         CClock::duration elapsed(now - lastFill);
         int64_t msElapsed = boost::chrono::duration_cast<boost::chrono::milliseconds>(elapsed).count();
-        if (msElapsed > 100) // note in practice msElapsed can be < 0, something to do with hyperthreading so reduce don't eliminate this conditional
+        if (msElapsed > 100) // note in practice msElapsed can be < 0, something to do with hyperthreading so reduce
+                             // don't eliminate this conditional
```

Comment:
While at it I'd say we could move the comment above the first line of code they are related to. i.e. I would avoid comment on the same line.